R/match_loanbooks.R

```diff
@@ -170,5 +170,12 @@ match_loanbooks <- function(config) {
       )
     cli::cli_progress_update()
   }
+  
   cli::cli_progress_done()
+  
+  write_manifest(
+    config = config,
+    path = file.path(output_matched_loanbooks_dir, "manifest.yml"),
+    prior_input_paths = c(output_prepare_dir)
```

Comment:
why not use `get_output_prepare_dir(config)` here, as is done for `analyse()`

Reply:
I questioned this too, but the output path, where the manifest should go, for any given function is already assigned to a variable and the function already used above, in this case `get_output_prepare_dir(config)`, so it seemed appropriate to use it rather than run the same function again?

Reply:
yea I guess. `match_loanbooks()` is a bit different in that it does not only call some lower level functions, but goes through the full process directly.. so I guess slightly different handling is appropriate